### PR TITLE
Allow text in feedback to include links.

### PR DESCRIPTION
**Pull request by seanlip**
This now contains tests and is ready for review.

### CODE REVIEW

**Comment by seanlip** [comment]
@eyurko could you PTAL at this? I updated it to reference the other PR that pulls out ThemeNameService.

Thanks!

File: client/question/question.js [hunk 1]
```diff
@@ -624,7 +625,8 @@ tie.constant('RUNTIME_ERROR_FEEDBACK_MESSAGES', {
         "Unfortunately Python doesn't support directly assigning ",
         "characters in a string. If you need to do so, try slicing the ",
         "string and adding new characters instead of assigning them. ",
-        "If you need a refresher on slicing, check out the primer."
+        "If you need a refresher on slicing, check out the ",
```
Comment (eyurko) [inline client/question/question.js]: wait this should be whatever we fixed to not be about slicing.

File: client/question/services/FeedbackGeneratorServiceSpec.js [hunk 2]
```diff
@@ -499,7 +499,8 @@ describe('FeedbackGeneratorService', function() {
             "Unfortunately Python doesn't support directly assigning ",
             "characters in a string. If you need to do so, try slicing the ",
             "string and adding new characters instead of assigning them. ",
-            "If you need a refresher on slicing, check out the primer."
+            "If you need a refresher on slicing, check out the ",
```
Comment (eyurko) [inline client/question/services/FeedbackGeneratorServiceSpec.js]: same here

File: client/question/question.js (hunk 1 above)
Comment (seanlip) [reply client/question/question.js]: I don't understand. I think this is the fixed version?

File: client/question/services/FeedbackGeneratorServiceSpec.js (hunk 2 above)
Comment (seanlip) [reply client/question/services/FeedbackGeneratorServiceSpec.js]: Ditto.

File: client/question/question.js (hunk 1 above)
Comment (eyurko) [reply client/question/question.js]: ah, you're right. Sorry, I got slicing and splicing confused; my mistake.

**Pull request merged**